app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -24,10 +24,14 @@ def question_navigator
         end
       end
 
+      def next_step
+        form_navigation.next
+      end
+
       def next_path
-        next_step = form_navigation.next
-        options = { us_state: params[:us_state], action: next_step.navigation_actions.first }
-        if next_step.resource_name.present? && next_step.resource_name == self.class.resource_name
+        step_for_next_path = next_step
+        options = { us_state: params[:us_state], action: step_for_next_path.navigation_actions.first }
+        if step_for_next_path.resource_name.present? && step_for_next_path.resource_name == self.class.resource_name
```

Comment:
i refactored this because it seems like we might want the resource_name logic here to run. all i really want to override is the next step, not necessarily the rest of the logic to construct the path. i think.